tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -72,6 +72,15 @@ et build //flutter/fml:fml_benchmarks  # Build a specific target in `//flutter/f
       allTargets.addAll(targets.map((target) => target.label));
     }
 
+    // Warn that we've discarded some targets.
+    // Other warnings should have been emitted above, so if this ends up being
+    // unneccesarily noisy, we can remove it or limit it to verbose mode.
+    if (allTargets.length < commandLineTargets.length) {
+      environment.logger.warning(
+        'One or more targets specified did not match any build targets.',
```

Comment:
It could be helpful to report at least one of the ones that didn't match, but I guess if folks are typing these in, there won't be so many that reporting *all* the ones that didn't match would be too much.

Reply:
That seems reasonable and easy to do! Done.